packages/kbn-storybook/index.js

```diff
@@ -62,21 +62,25 @@ exports.runStorybookCli = config => {
         // route errors
         subj.toPromise(),
 
-        new Promise(() => {
+        new Promise(async () => {
           // storybook never completes, so neither will this promise
           const configDir = join(__dirname, 'storybook_config');
           log.debug('Config dir:', configDir);
-          storybook({
-            mode: 'dev',
+          await storybook({
+            mode: flags.site ? 'static' : 'dev',
             port: 9001,
             configDir,
+            outputDir: flags.site ? `${REPO_ROOT}/storybook-build` : undefined,
```

Comment:
Maybe we can add here `name` to allow different builds to co-exist. And build it in Storybook folder so that `yarn storybook --clean` deletes static Storybook builds.

```
join(ASSET_DIR, name);
```

`ASSET_DIR` can be imported from `./lib/constants`.

Reply:
Than we can build Storybooks for all plugins and upload them to some storage from CI.